COCO_DATA_FORMAT.md

```diff
@@ -227,4 +227,26 @@ people_dataset/
     train_coco_location.json
     test_coco_location.json
 ```
-```
\ No newline at end of file
+
+
+## KeyValuePair dataset
+
+KeyValuePair dataset is associated with a schema defining a task that input is text + (multi-)image, label is key-value pairs. Each sample of dataset consists of
+1. images input, each image is optionally associated with text metadata as a dictionary
```

Comment:
probably clearer:
- input:
    - images
    - text
- output:
    - a dictionary with keys being the attributes of interest

One thing is to mention whether images or text can be optional
One thing is whether you would want text**(s)** input